### DISCUSSION TITLE
Anyway to listen context.globalState changes?

### DISCUSSION THREAD

**Discussion by sudoaugustin** [microsoft/vscode-discussions#34] [Extension Development QnA] [answered]
How do I know **synced** `context.globalState`  got an update? Need to perform `fs.write` on it. Also can't use globalStoragePath as a solution.

**Comment by alefragnani** [comment, marked as answer]
There is no API available for that. 

I suggest you to open a Feature Request in VS Code repo.

**Reply by alefragnani** [reply]
BTW, it seems an _event_ was planned, but for later.

https://github.com/microsoft/vscode/issues/95209#issuecomment-716490477

**Reply by sudoaugustin** [reply]
Here
https://github.com/microsoft/vscode/issues/152238#issue-1272581323
